doc/dnf5.conf.5.rst

```diff
@@ -741,6 +739,12 @@ configuration.
 
     Default: ``75``
 
+.. _disable_excludes_options-label:
+``disable_excludes``
+    :ref:`list <list-label>`
+
+    Used to disable packages filtering. It can include a list of repository IDs (globs allowed) for which repository-specific excludes are ignored. If it contains the special value ``main`` excludes from the main configuration files are bypassed. If it contains the special value ``*`` no excludes filtering is applied at all.
```

Comment:
Please add commas after the conditional sentences: "value ``main``, excludes" and "value ``*``, no excludes".